### DISCUSSION TITLE
Trace to Logs functionality for Elasticsearch Datasource

### DISCUSSION THREAD

**Discussion by aengusrooneygrafana** [grafana/grafana#51746] [Data Sources]
We would like to have the same 'Trace to Logs' functionality for the Elasticsearch datasource as is currently available in the Loki and Splunk plugins (described here - https://grafana.com/docs/grafana/latest/datasources/tempo/#trace-to-logs).  With support for the latest version of Elastic, plus two older versions if possible?

**Comment by ivanahuckova** [comment]
This feature was implemented.
